### PR TITLE
Wip/udayan/resultactions

### PR DESCRIPTION
**Pull request by Udayan12167**

### CODE REVIEW

File: coalib/coala.py [hunk 1]
```diff
@@ -22,7 +26,7 @@
 
 def main():
     log_printer = ConsolePrinter()
-    interactor = ConsoleInteractor(log_printer)
+    console_printer = ConsolePrinter()
```
Comment (AbdealiJK) [inline coalib/coala.py]: Can't we use the same consolePrinter for log_printer ?

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+
+
+def format_line(line, real_nr="", sign="|", mod_nr="", symbol="", ):
+    return "|{:>4}{}{:>4}|{:1}{}".format(real_nr,
+                                         sign,
+                                         mod_nr,
+                                         symbol,
+                                         line.rstrip("\n"))
+
+
```
Comment (AbdealiJK) [inline coalib/output/ConsoleInteraction.py]: Extra newline

File: coalib/coala.py (hunk 1 above)
Comment (Udayan12167) [inline coalib/coala.py]: In the future we want the possibility to give the user an option of having log_printers which can print to files.

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**